Sources/Base64/Base64.swift

```diff
@@ -59,51 +56,165 @@ extension Base64 {
   
   // MARK: Internal
   
-  // The base64 unicode table.
-  @usableFromInline
-  static let encodeBase64: [UInt8] = [
-     65,  66,  67,  68,  69,  70,  71,  72,
-     73,  74,  75,  76,  77,  78,  79,  80,
-     81,  82,  83,  84,  85,  86,  87,  88,
-     89,  90,  97,  98,  99, 100, 101, 102,
-    103, 104, 105, 106, 107, 108, 109, 110,
-    111, 112, 113, 114, 115, 116, 117, 118,
-    119, 120, 121, 122,  48,  49,  50,  51,
-     52,  53,  54,  55,  56,  57,  43,  47,
-  ]
-  
-  @usableFromInline
-  static let encodeBase64Url: [UInt8] = [
-     65,  66,  67,  68,  69,  70,  71,  72,
-     73,  74,  75,  76,  77,  78,  79,  80,
-     81,  82,  83,  84,  85,  86,  87,  88,
-     89,  90,  97,  98,  99, 100, 101, 102,
-    103, 104, 105, 106, 107, 108, 109, 110,
-    111, 112, 113, 114, 115, 116, 117, 118,
-    119, 120, 121, 122,  48,  49,  50,  51,
-     52,  53,  54,  55,  56,  57,  45,  95,
-  ]
+
   
   @usableFromInline
   static let encodePaddingCharacter: UInt8 = 61
+
+  // The base64 unicode table.
+  @inlinable
+  static func encodeLookup(byte: UInt8) -> UInt8 {
+    switch byte {
```

Comment:
@fabianfett try

```
case 0...51:
    return byte + 65
case 52..63:
     return byte - 4
default:
     fatalError(..)
}
```

Reply:
@weissi I updated the code to the proposed solution. It's a little faster than before (though that might be measuring error - not sophisticated enough perf tests), but it is still considerably slower than the lookup table (about 2x).

See https://github.com/fabianfett/swift-base64/pull/3/checks

Reply:
@fabianfett that's interesting. I just looked at the [generated code](https://godbolt.org/z/7TF5yf) for the `switch` and it's actually not great. Usually LLVM manages to optimise the hell out of those switches, not in this case it seems.

Reply:
@fabianfett mind giving this a shot?

```
func encodeLookup(byte: UInt8) -> UInt8 {
    let table = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_".utf8
    return table[table.index(table.startIndex, offsetBy: Int(byte))]
}
```

This shouldn't do any ARC whatsoever so it might be quite fast. If not we can file a Swift bug.